ivy/functional/backends/jax/creation.py

```diff
@@ -92,13 +92,20 @@ def eye(
     n_rows: int,
     n_cols: Optional[int] = None,
     k: Optional[int] = 0,
+    batch_shape: Optional[Union[int, Tuple[int], List[int]]] = None,
```

Comment:
we should use `Sequence[int]` type hint instead of `Tuple[int]` and `List[int]`

Reply:
Yes I am working on that. Thanks!